openapi/go.sh

```diff
@@ -43,7 +43,7 @@ pushd "${OUTPUT_DIR}" > /dev/null
 OUTPUT_DIR=`pwd`
 popd > /dev/null
 
-source "${SCRIPT_ROOT}/swagger-codegen/client-generator.sh"
+source "${SCRIPT_ROOT}/openapi-generator/client-generator.sh"
 source "${SETTING_FILE}"
 
 SWAGGER_CODEGEN_COMMIT="${SWAGGER_CODEGEN_COMMIT:-v2.3.0}"; \
```

Comment:
This line can be removed or replaced by https://github.com/kubernetes-client/gen/blob/c05fbc861bd7827bd88fd9b910b6eae04382d235/openapi/openapi-generator/client-generator.sh#L44

Reply:
Done.